### ISSUE TITLE
Exception in tflite with missing operators that model uses

### ISSUE THREAD

**Issue by badrulaiman53** [tensorflow/tensorflow#32427]
**System information**
- OS Platform and Distribution : Ubuntu 18.04.2 LTS
- TensorFlow installed from binary
- TensorFlow version tf-1.0.0 (or github SHA if from source):

Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: AVERAGE_POOL_2D, CAST, CONCATENATION, CONV_2D, EXPAND_DIMS, FULLY_CONNECTED, MAX_POOL_2D, RESIZE_BILINEAR, SOFTMAX. Here is a list of operators for which you will need custom implementations: DecodeJpeg.

link to a GraphDef or the model.
https://github.com/RaghavPrabhu/Deep-Learning/tree/master/dogs_breed_classification

**Comment by mihaimaruseac** [comment]
Note that TF 1.0.0 is extremely old and not supported

**Comment by alanchiao** [comment]
As noted in the error message, DecodeJpeg (https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/decode-jpeg) has not been added to TFLite yet. Consider adding this as a [custom op](https://www.tensorflow.org/lite/guide/ops_custom) and perhaps then contributing it as a builtin operator.

**Comment by jdduke** [comment]
We don't have any immediate plans to add this as a builtin-op. However, you should be able to convert your model *after* the decode operation, and use platform-specific functionality to load the jpeg into memory and pass as an input into the tensor.
